resources/assets/sass/app.scss

```diff
@@ -310,6 +316,9 @@ body, html {
 
 .nav>li>a:focus, .nav>li>a:hover {
   background-color: #00b0f0;
+  -webkit-transition: background-color 0.2s, color 0.2s, -webkit-transform 0.2s;
+  transition: background-color 0.2s, color 0.2s, transform 0.2s;
+  -webkit-transition-timing-function: ease-out;
```

Comment:
This only exists with the vendor prefix - there's no transition-timing-function property listed

Vendor prefix can probably just be removed, as with line 319